omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -428,21 +428,16 @@
 default['private_chef']['lb']['redis_connection_timeout'] = 1000
 default['private_chef']['lb']['redis_keepalive_timeout'] = 2000
 default['private_chef']['lb']['redis_connection_pool_size'] = 250
-default['private_chef']['lb']['maint_refresh_interval'] = 600
-default['private_chef']['lb']['ban_refresh_interval'] = 600
+default['private_chef']['lb']['maint_refresh_interval'] = 600 # the data from redis to the shared memory in lua may take up to 600 sec
+default['private_chef']['lb']['ban_refresh_interval'] = 600 # not used
 default['private_chef']['lb']['chef_min_version'] = 10
 default['private_chef']['lb']['access_by_lua_file'] = false
 
 ###
 # Load balancer route configuration
 ###
 default['private_chef']['lb']['xdl_defaults']['503_mode'] = false
-default['private_chef']['lb']['xdl_defaults']['couchdb_containers'] = false
```

Comment:
would removing this cause issues with chef-mover in migration from an older version? It also has some defaults in the hab plan.sh Which should be removed as well. I don't mind doing the cleanup as a part of a separate PR, if that works better.

Reply:
Will add it in a clean up PR